lib/benchee/benchmark/runner.ex

```diff
@@ -33,10 +33,10 @@ defmodule Benchee.Benchmark.Runner do
                           scenario_context = %ScenarioContext{printer: printer, config: config}) do
     printer.input_information(input_name, config)
     printer.benchmarking(job_name, config)
-    pmap 1..config.parallel, fn ->
+    Parallel.map(1..config.parallel, fn(_) ->
```

Comment:
I think if we just did `_task_number` it'd be fine, I mean the collection is created right in the statement before so it's fairly obvious that it's a range - or so I think :)